src/ui/toast.js

```diff
@@ -22,12 +22,8 @@ import {
 
 /** @const {!Object<string, string|number>} */
 export const toastImportantStyles = {
-  'position': 'fixed',
-  'bottom': 0,
   'height': 0,
   'max-height': '46px',
```

Comment:
Why not put `max-height` in CSS as well? Are you updating it dynamically as well?

Reply:
Removed from here since already included in CSS